bigbone-rx/src/main/kotlin/social/bigbone/rx/RxFeaturedTagsMethods.kt

```diff
@@ -21,57 +20,26 @@ class RxFeaturedTagsMethods(client: MastodonClient) {
      * List all hashtags featured on your profile.
      * @return List of [FeaturedTag]s on your profile
      */
-    fun getFeaturedTags(): Single<MastodonRequest<List<FeaturedTag>>> {
-        return Single.create {
-            try {
-                it.onSuccess(featuredTagsMethods.getFeaturedTags())
-            } catch (e: Throwable) {
-                it.onError(e)
-            }
-        }
-    }
+    fun getFeaturedTags(): Single<List<FeaturedTag>> =
```

Comment:
This was wrong before. I forgot to call `.execute()` and thus returned a `Single<MastodonRequest<List<FeaturedTags>>>` instead of `Single<List<FeaturedTags>>`.

I have fixed this issue at the same time as rewriting it for the main change of this PR.